yt_dlp/extractor/fujitv.py

```diff
@@ -5,44 +5,67 @@
 
 
 class FujiTVFODPlus7IE(InfoExtractor):
-    _VALID_URL = r'https?://fod\.fujitv\.co\.jp/title/[0-9a-z]{4}/(?P<id>[0-9a-z]+)'
-    _BASE_URL = 'http://i.fod.fujitv.co.jp/'
-    _BITRATE_MAP = {
-        300: (320, 180),
-        800: (640, 360),
-        1200: (1280, 720),
-        2000: (1280, 720),
-        4000: (1920, 1080),
-    }
+    _VALID_URL = r'https?://fod\.fujitv\.co\.jp/title/(?P<sid>[0-9a-z]{4})/(?P<id>[0-9a-z]+)'
+    _BASE_URL = 'https://i.fod.fujitv.co.jp/'
+    # _BITRATE_MAP = {
+    #     300: (320, 180),
+    #     800: (640, 360),
+    #     1200: (1280, 720),
+    #     2000: (1280, 720),
+    #     4000: (1920, 1080),
+    #     6000: (1920, 1080),
+    # }
```

Comment:
Don't leave commented code. If it's not used, delete it. If it's stuff to fix, mark PR as draft